EDDI/MainWindow.xaml.cs

```diff
@@ -858,50 +858,32 @@ public static void prepareLog(IProgress<string> progress)
                 string issueLogFile = issueLogDir + "eddi_issue.log";
                 string desktopPath = Environment.GetFolderPath(Environment.SpecialFolder.Desktop) + @"\eddi_issue.zip";
 
-                string[] log;
+                // Create a temporary issue log file, delete any remnants from prior issue reporting
                 lock (logLock)
                 {
-                    log = File.ReadAllLines(Constants.DATA_DIR + @"\eddi.log");
+                    Directory.CreateDirectory(issueLogDir);
+                    File.WriteAllText(issueLogFile, File.ReadAllText(Constants.DATA_DIR + @"\eddi.log"));
                 }
-                if (log.Length == 0) { return; }
-
-                progress.Report("");
-                List<string> outputLines = new List<string>();
-                // Use regex to isolate DateTimes from the string
-                Regex recentLogsRegex = new Regex(@"^(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2})");
 
-                foreach (string line in log)
+                // Truncate log files more than the specified size MB in size
+                long maxLogSizeMB = 2;
+                long maxLogSizeBytes = maxLogSizeMB * (long)Math.Pow(1024, 2);
```

Comment:
A hexadecimal constant is probably better here.

Reply:
Why is that? Expressing it as we are, it is quite readable and understandable.

Reply:
Because Pow() is in general quite expensive and I'm not sure whether the compiler has the smarts to recognize a power of 2 and substitute the correct value at compile-time.

Reply:
A compile-time constant is just as readable and understandable with the bonus of being a lot more effiicient.